lib/routes/account.js

```diff
@@ -1113,13 +1115,21 @@ module.exports = function (
               push.notifyUpdate(uid, 'accountVerify')
 
               return db.verifyEmail(account)
-                .then(mailer.sendPostVerifyEmail.bind(
-                    mailer,
-                    account.email,
-                    {
-                      acceptLanguage: request.app.acceptLanguage
+                .then(
+                  function() {
+                    // Our post-verification email is very specific to sync,
+                    // so don't send it if we're sure this is not for sync.
+                    // Older clients will not send a 'service' param here
```

Comment:
I could say "older versions of the content-server" to be more clear about that...